src/Lib/Mail/Mail.php

```diff
@@ -0,0 +1,105 @@
+<?php
+declare(strict_types = 1);
+
+namespace App\Lib\Mail;
+
+class Mail
+{
+
+    /** @var string */
+    private $headders;
+
+
+    /**
+     *
+     * @param string $to
+     * @param string $from
+     * @param string $fromName
+     * @param string $content
+     */
+    public function __construct(string $to, string $from, string $fromName, string $subject, string $content)
+    {
+        $this->to = $to?$to:false;
+        $this->from = testEmail($from, 'From')?$from:false;
+        $this->fromName = $fromName?$fromName:false;
+        $this->subject = $subject?$subject:false;
+        $this->content = $content?$content:false;
+    }
+
+
+    /**
+    * Add hidden copy
+    *
+    * @return void
+    */
+    public static function addBcc($bcc):void
+    {
+        $this->bcc = testEmail($bcc, 'Bcc')?$bcc:false;
+    }
+
+
+    /**
+    * Add reply to (different address)
+    *
+    * @return void
+    */
+    public static function addReplyTo($replyTo, $replyToName):void
+    {
+        $this->replyTo = testEmail($replyTo, 'Reply-to')?$replyTo:false;
+        $this->replyToName = $replyToName;
+    }
+
+
+    /**
+     * Test for email address
+     * 
+     * @return bool
+     */
+    private static function testEmail(string $email, string $case):bool
```

Comment:
Maybe rename to assertEmail? Or better make new lib to test these kind of strings.

Reply:
Nice idea!
So: Lib for testing stuff? For example ./Lib/Testers.php:
test mail address, test if date, test if special symbols, etc.?

Reply:
Yes, we can have /Lib/Assert and inside  String.php and method isEmail("email"). So it will be used as use `App\Lib\Assert\String; String->isEmail('fooo');`.